### ISSUE TITLE
Add bigger logo to home page

### ISSUE THREAD

**Issue by domoritz** [uwdb/website#18]
@stechu suggested that we use a larger version of the logo on the home page. We should use the white version. Get it from https://github.com/uwdb/logo

Prefer svg.

**Comment by stechu** [comment]
@domoritz  or gold?
https://github.com/uwdb/logo/blob/master/uwdb_monogold_transparent.svg

White transparent on top of nearly white background does not seem to be a good idea to me.

**Comment by domoritz** [comment]
The background should be dark enough so that white works. Let's not add too many colors. I think the homepage already has too many colors.

**Comment by stechu** [comment]
@domoritz  @maaz139  do we have a purple logo? It seems to break the design a lot if we change to dark background.

**Comment by domoritz** [comment]
I thought we want to put the logo here:

![screen shot 2015-03-17 at 14 55 28](https://cloud.githubusercontent.com/assets/589034/6698438/e973eb94-ccb5-11e4-8852-c31179771945.png)

I think white works. Can you send a screenshot?

**Comment by stechu** [comment]
oh, that makes sense. 
I was thinking to make the navagation bar wider.

**Comment by domoritz** [comment]
Sorry for not being clear about this. I thought that what you meant.

**Comment by akcheung** [comment]
I agree with @domoritz about the white logo. Should we give it a try?
